src/thunderbots/software/test/ai/hl/stp/action/kick_action.cpp

```diff
@@ -136,7 +136,7 @@ TEST(KickActionTest, robot_not_behind_ball_kicking_towards_positive_x_positive_y
         MoveIntent move_intent = dynamic_cast<MoveIntent &>(*intent_ptr);
```

Comment:
Why don't we catch `std::bad_cast` here like we do in the `chip_action` tests?

Reply:
Is this separate from your above comment? It is doing a try-catch, just not the specific exception. See my reply above.